### PR TITLE
Remove owner role assignments from live test ARM templates

### PR DESCRIPTION
**Pull request by benbp**
- Remove owner role assignment from time series insights ARM template
- Remove owner role assignment from digital twins ARM template
- - Remove owner role assignment from iot ARM template

[Recent updates](https://github.com/Azure/azure-sdk-tools/blob/b9c69e2f593b375552d47acd0ba5e650d5fdd005/eng/common/TestResources/New-TestResources.ps1#L466-L475) to the `New-TestResources.ps1` script will do this assignment on deployment already. It is creating `role assignment already exists` errors for these templates.

FYI @jsquire @heaths @weshaggard @azabbasi

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]
LGTM assuming tests pass.

File: sdk/digitaltwins/test-resources.json
Comment (heaths) [inline sdk/digitaltwins/test-resources.json:8]: If they have a bicep file, should the JSON file even be checked in? Unless used for something else in their workflow, it's otherwise redundant.

**Review by heaths — APPROVED** [review]
Probably not a concern, but did you make sure nothing has a dependency on these?

File: sdk/digitaltwins/test-resources.json
Comment (azabbasi) [reply sdk/digitaltwins/test-resources.json:8]: We have a dev side setup script that runs the ARM template, but we can use the same bicep builder to generate it and wipe it out afterwards, I will amend that change after this is merged in.

**Comment by azabbasi** [comment]
The only drawback to this is when the ARM template is being deployed outside of the New-TestResources script and the assumption of ownership is invalid. I will adjust the local setup script to fix that issue as well.

**Review by azabbasi — APPROVED** [review]

File: sdk/digitaltwins/test-resources.json
Comment (heaths) [reply sdk/digitaltwins/test-resources.json:8]: If it's getting used, not a big deal. Just wanted to make sure it wasn't committed in err.

Comment (benbp) [reply sdk/digitaltwins/test-resources.json:8]: @azabbasi the existing `az` and powershell commands to deploy templates can take a bicep template as input, if your dev side scripts use those. The only thing missing is build+local validation.

**Pull request merged**
